### PR TITLE
Adding eap64-basic-s2i.json and updating eap64-mysql-persistent-s2i

### LINKED ISSUE #99: Add eap64-basic-s2i.json template
We have "eap64-mysql-persistent-s2i"  already packaged in adb-utils. Refer https://github.com/projectatomic/adb-utils/issues/13#issuecomment-188722149

 We also need https://github.com/jboss-openshift/application-templates/blob/master/eap/eap64-basic-s2i.json as it a simpler application then eap64-mysql-persistent-s2i and takes less time to start.

CC @hferentschik

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Added eap64-basic-s2i.json and updated eap64-mysql-persistent-s2i
OpenShift templates as per the latest release
https://github.com/jboss-openshift/application-templates/tree/ose-v1.3.0/eap

This PR Fixes #99

Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

File: services/openshift/scripts/openshift_provision [hunk 1]
```diff
@@ -129,6 +129,7 @@ if [ ! -f ${ORIGIN_DIR}/configured.templates ]; then
 
     #EAP
     /opt/adb/openshift/templates/eap64-mysql-persistent-s2i.json
+    /opt/adb/openshift/templates/eap64-basic-s2i.json
```
Comment (hferentschik) [inline services/openshift/scripts/openshift_provision:132]: +1 good to see it comes back. Please make sure to get this merged prior code freeze.

File: services/openshift/scripts/openshift_provision [hunk 2]
```diff
@@ -129,6 +129,7 @@ if [ ! -f ${ORIGIN_DIR}/configured.templates ]; then
 
     #EAP
     /opt/adb/openshift/templates/eap64-mysql-persistent-s2i.json
```
Comment (hferentschik) [inline services/openshift/scripts/openshift_provision:131]: Any chance you verified it works? Ephemeral better!?

File: services/openshift/templates/eap64-mysql-persistent-s2i.json [hunk 3]
```diff
@@ -446,7 +446,7 @@
                         }
                     },
                     "spec": {
-                        "serviceAccount": "eap-service-account",
+                        "serviceAccountName": "eap-service-account",
```
Comment (hferentschik) [inline services/openshift/templates/eap64-mysql-persistent-s2i.json:449]: ooi, what is this about?

**Comment by hferentschik** [comment]
+1 to getting the basic EAP back. Definitely integrate. I would love to know though as well whether anyone can confirm the persistent EAP + MySQL works in a reasonable matter as well. Mind you, it is not only about getting it just to build. Persistent also implies to me, that the db data can be kept across restarts of the VM and even full destroy/re-provision cycles. Right now the data is gone on restart or latest after destroying the VM. For this reason I think ephemeral sends the more correct message.

File: services/openshift/templates/eap64-mysql-persistent-s2i.json (hunk 3 above)
Comment (praveenkumar) [inline services/openshift/templates/eap64-mysql-persistent-s2i.json:449]: Looks like it changed https://github.com/jboss-openshift/application-templates/commit/373e1de465c58018c129f85229be83f7d047499b#diff-cbe42c8c682826a475a22ba56a93ae81 so @LalatenduMohanty just  updated it (But we have to make sure that is also work with ose-3.1.1.6 which we are going to ship with CDK).

File: services/openshift/scripts/openshift_provision (hunk 2 above)
Comment (LalatenduMohanty) [inline services/openshift/scripts/openshift_provision:131]: @hferentschik sorry did not get you? Are you suggesting for a name change or something?
I have not verified whether eap64-mysql-persistent-s2i works. I guess once we have build ready we will ask relevant users to test it out.

File: services/openshift/templates/eap64-mysql-persistent-s2i.json (hunk 3 above)
Comment (LalatenduMohanty) [inline services/openshift/templates/eap64-mysql-persistent-s2i.json:449]: Thats right, I just borrowed the change from upstream

**Comment by praveenkumar** [comment]
LGTM

**Pull request merged**